layouts/partials/scripts.html

```diff
@@ -1,3 +1,5 @@
-<script src="{{ .Site.BaseURL }}assets/scripts/main.js"></script>
+<!-- Uncomment once we have some actual code to include
+    <script src="{{ .Site.BaseURL }}assets/scripts/main.js"></script>
+-->
```

Comment:
Should we document how the `main.js` file gets generated from the `start.js` that is being deleted? Or just delete that functionality from the gulp file all together?